examples/suit_update/README.md

```diff
@@ -0,0 +1,558 @@
+from the newest image.
+
+#### Key Generation
+
+To sign the manifest and for the device to verify the manifest a pair of keys
+must be generated. Note that this is done automatically when building an
+updatable RIOT image with `riotboot` or `suit/publish` make targets.
+
+This is simply done using the `suit/genkey` make target:
+
+    $ BOARD=samr21-xpro make -C examples/suit_update suit/genkey
+
+You will get this message in the terminal:
+
+    Generated public key: 'a0fc7fe714d0c81edccc50c9e3d9e6f9c72cc68c28990f235ede38e4553b4724'
+
+### Network
+
+For connecting the device with the internet we are using ethos (a simple
+ethernet over serial driver).
+
+When executing $RIOTBASE/dist/tools/ethos:
+
+    $ sudo ./start_network.sh /dev/ttyACM0 riot0 2001:db8::1/64
+
+A tap interface named `riot0` is setup. `fe80::1/64` is set up as it's
+link local address and `fd00:dead:beef::1/128` as the "lo" unique link local address.
+
+Also `2001:db8::1/64` is configured- as a prefix for the network. It also sets-up
+a route to the `2001:db8::1/64` subnet through `fe80::2`. Where `fe80::2` is the default
+link local address of the UHCP interface.
+
+Finally when:
+
+    $ sudo ip address add 2001:db8::1/128 dev riot0
+
+We are adding a routable address to the riot0 tap interface. The device can
+now send messages to the the coap server through the riot0 tap interface. You could
+use a different address for the coap server as long as you also add a routable
+address, so:
+
+    $ sudo ip address add $(SUIT_COAP_SERVER) dev riot0
+
+When using a border router the same thing is happening although the node is no
+longer reachable through its link local address but routed through to border router
+so we can reach it with its global address.
+
+NOTE: if we weren't using a local server you would need to have ipv6 support
+on your network or use tunneling.
+
+NOTE: using `fd00:dead:beef::1` as an address for the coap server would also
+work and you wouldn't need to add a routable address to the tap interface since
+a route to the loopback interface (`lo`) is already configured.
+
+### Server and file system variables
+
+The following variables are defined in makefiles/suit.inc.mk:
+
+    SUIT_COAP_BASEPATH ?= firmware/$(APPLICATION)/$(BOARD)
+    SUIT_COAP_SERVER ?= localhost
+    SUIT_COAP_ROOT ?= coap://$(SUIT_COAP_SERVER)/$(SUIT_COAP_BASEPATH)
+    SUIT_COAP_FSROOT ?= $(RIOTBASE)/coaproot
+    SUIT_PUB_HDR ?= $(BINDIR)/riotbuild/public_key.h
+
+The following convention is used when naming a manifest
+
+    SUIT_MANIFEST ?= $(BINDIR_APP)-riot.suitv4.$(APP_VER).bin
+    SUIT_MANIFEST_LATEST ?= $(BINDIR_APP)-riot.suitv4.latest.bin
+    SUIT_MANIFEST_SIGNED ?= $(BINDIR_APP)-riot.suitv4_signed.$(APP_VER).bin
+    SUIT_MANIFEST_SIGNED_LATEST ?= $(BINDIR_APP)-riot.suitv4_signed.latest.bin
+
+The following default values are using for generating the manifest:
+
+    SUIT_VENDOR ?= RIOT
+    SUIT_VERSION ?= $(APP_VER)
+    SUIT_DEVICE_ID ?= $(BOARD)
```

Comment:
I would change ` SUIT_DEVICE_ID` by `SUIT_CLASS_ID` or `SUIT_DEVICE_CLASS_ID` in case `SUIT_VALIDATED_DEVICE` is added in the future.

Reply:
done